net-proxy/trojan/trojan-1.7.0.ebuild

```diff
@@ -0,0 +1,35 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+inherit cmake-utils
+
+DESCRIPTION="An unidentifiable mechanism that helps you bypass GFW"
+HOMEPAGE="https://github.com/trojan-gfw/trojan"
+SRC_URI="https://github.com/trojan-gfw/${PN}/archive/v${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="GPL-3"
+SLOT="0"
+KEYWORDS="amd64 x86 arm arm64"
```

Comment:
Don't add new ebuilds with stable keywords. Add only `~arch` keywords, and only for arches you've actually physically tested on. Also don't add `~arm` without contacting ARM arch team first since they have pretty solid requirements on testing on different ARM versions.